sensing/autoware_pointcloud_preprocessor/src/distortion_corrector/distortion_corrector_node.cpp

```diff
@@ -91,9 +92,27 @@ void DistortionCorrectorComponent::onPointCloud(PointCloud2::UniquePtr pointclou
   }
 
   distortion_corrector_->setPointCloudTransform(base_frame_, pointcloud_msg->header.frame_id);
-
   distortion_corrector_->initialize();
-  distortion_corrector_->undistortPointCloud(use_imu_, *pointcloud_msg);
+
+  if (update_azimuth_and_distance_ && !can_update_azimuth_and_distance_) {
+    can_update_azimuth_and_distance_ =
+      distortion_corrector_->azimuthConversionExists(*pointcloud_msg);
+    if (can_update_azimuth_and_distance_) {
+      RCLCPP_INFO(
+        this->get_logger(),
+        "Success to get the conversion formula between cartesian coordinates and LiDAR azimuth "
+        "coordinates");
+    } else {
+      RCLCPP_ERROR(
+        this->get_logger(),
+        "Failed to get the conversion formula between cartesian coordinates and LiDAR azimuth "
+        "coordinates. Please check the LiDAR azimuth coordinate. This pointcloud will not update "
+        "azimuth and distance");
```

Comment:
Stopping execution here is imho better because the user configured the node wrong. Trying to continue execution imo is not particularly meaningful.

Reply:
True, add the runtime error in https://github.com/autowarefoundation/autoware.universe/pull/8380/commits/7e1f510b89f2b0425547cef9ff85b228f1f62437. 
Thanks a lot!

Reply:
Thanks! fixed in https://github.com/autowarefoundation/autoware.universe/pull/8380/commits/1d5ccdb1c8b6717ca20bd527f298b218dcdb3be5